CHANGES.rst

```diff
@@ -8,7 +8,7 @@ Improvements:
  - Any Account data element, even if the type is not known is persisted.
 
 Bugfix:
- -
+ - Room members who left are listed with the actual members (vector-im/riot-android#2744)
```

Comment:
Note: this PR fix "I'm not allow to send message in a new joined room" #2743 too